README.md

```diff
@@ -15,12 +20,13 @@ Paste is possible from a filetype to another.
 
 Current supported filetypes:
 * Verilog
+* SystemVerilog
 * VHDL
 
 
 ### Commands
 `:VlsiYank                            ` Yank the module containing the cursor.
-                                       default mapping: `<M-F6>`
+                                       default mapping: `<M-F6>`, `<leader-y>`
```

Comment:
This is not updated from [autoload / vlsi.vim](https://github.com/ythonnart/vim-vlsi/pull/1/files#diff-3e098e1f3eb1bba00b4c28baf8a1e8fceef16cb3aed28b1408468acced58a68a)
where you swapped <F6> and <M-F6> for VlsiYank and VlsiPasteAsInstance.
I was in favor of using <F6> to paste as instance because pasting multiple instances is more frequent than any other Yank/Paste operation, so I’d rather keep the old default mappings

Reply:
will revert to previous mapping 👍